src/debug/HyprCtl.hpp

```diff
@@ -29,4 +29,11 @@ class CHyprCtl {
     wl_event_source*                 m_eventSource = nullptr;
 };
 
+namespace HyprCtl {
+    std::string getWindowData(PHLWINDOW w, eHyprCtlOutputFormat format);
```

Comment:
why not put them inside CHyprCtl? It's a singleton. If you prefer, you can even make these static.

Reply:
Aight done chief :saluting_face: . Should I just add every method into the class and make it static in case someone else needs it in the future, or you want it not to be more cluttered than needed?